guides/v2.3/ui_comp_guide/components/ui-secondary-uiselect.md

```diff
@@ -90,15 +90,15 @@ Navigation keys:
     <td><code>imports</code></td>
     <td>Defines from where the component receives its data</td>
     <td>
-    {% highlight javascript %}imports: {
+    ```javascriptimports: {
```

Comment:
Just `javascript` not `javascriptimports` right?

Reply:
correct